process_report/processors/validate_pi_alias_processor.py

```diff
@@ -1,12 +1,13 @@
-from dataclasses import dataclass
+from dataclasses import dataclass, field
 
+from process_report.config import config
 from process_report.invoices import invoice
 from process_report.processors import processor
 
 
 @dataclass
 class ValidatePIAliasProcessor(processor.Processor):
-    alias_map: dict
+    alias_map: dict = field(default_factory=config.get_alias_map)
```

Comment:
Generally, we expect to see type arguments, like `dict[str, str]`, rather than an untyped `dict`.